CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Is it possible to have a lower version of cmake?  What I generally do is to target the version of CMake that is in the oldstable version of Debian(as of right now, Debian 9(stretch) has cmake 3.7).  This is not a requirement.

Reply:
I think I may have used some cmake that requires 3.13. It was 6 months ago so I cannot recall exact details.

I have been trying to follow best practices in Professional CMake (4th Edition)